cmd/agent/main.go

```diff
@@ -11,31 +12,59 @@ import (
 	"github.com/AlekseyKas/metrics/internal/storage"
 )
 
+var (
+	// Build version.
+	buildVersion string
```

Comment:
```suggestion
	buildVersion = "N/A"
```
и тогда не нужны if на 52 строке